app/assets/v2/css/base.css

```diff
@@ -652,7 +652,7 @@ input[type=text].loading {
   color: #4aeba8;
 }
 
-.submit_bounty select[name=deonomination] {
+.submit_bounty select[name=denomination] {
```

Comment:
You'll want to update your master branch and this PR branch from our upstream to remove these previously introduced line changes.